rpm/SOURCES/cntlm-user

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# cNTLM per user launcher script
+USERNAME="$(id -un)"
+HOME="$(getent passwd "$(id -u)" | cut -d: -f6)"
+USER_CONF_DIR="${HOME}/.cntlm"
+
+if [ ! -e "${USER_CONF_DIR}" ] ; then
+  echo "${USER_CONF_DIR} does not exist! Unable to run cNTLM for user ${USERNAME}."
```

Comment:
What if the directory exists, but not the configuration file?

Reply:
There should be XDG related environment variables to detect the configuration directory btw. Nowadays it's typically a level deeper (below `.config` or `.local`).

Reply:
Both valid points.

Addressed in manner adhering to https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html in commit 091787a64561b84cf765ae4e09ce73cbdf380ea4